docs/tutorials/resource-migration.md

```diff
@@ -0,0 +1,235 @@
+
+After the above command executed, you will see Karmada control plane installed with multi members clusters.
+
+### Enable PropagationPolicyPreemption in karmada-controller-manager
+
+**Step 2: Run the command**
+
+```shell
+$ kubectl --context karmada-host get deploy karmada-controller-manager -n karmada-system -o yaml | sed '/- --failover-eviction-timeout=30s/{n;s/- --v=4/- --feature-gates=PropagationPolicyPreemption=true\n        &/g}' | kubectl --context karmada-host replace -f -
+```
+
+**Note:**
+
+The feature `PropagationPolicy Priority and Preemption` was introduced in v1.7, and it is controlled by the feature gate `PropagationPolicyPreemption` which is disabled by default.
+
+You can just execute the above one command to enable this feature gate. Or, if you want to use a more cautious approach, you can do like this:
+
+1. execute `kubectl --context karmada-host edit deploy karmada-controller-manager -n karmada-system` 
+2. check if feature gate `--feature-gates=PropagationPolicyPreemption=true` is existed in `spec.template.spec.containers[0].command` field.
+3. If not, you shall add `--feature-gates=PropagationPolicyPreemption=true` into that field.
+
+### Preset resource in a member cluster
+
+To simulate resources already exist in the member cluster, we deploy some simple Deployments and Services to `member1` cluster.
+
+**Step 3: Write the code**
+
+Create new file `/tmp/deployments-and-services.yaml` and copy text below to it:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deploy
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:latest
+        ports:
+        - containerPort: 80
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx-svc
+spec:
+  selector:
+    app: nginx
+  type: NodePort
+  ports:
+  - port: 80
+    nodePort: 30000
+    targetPort: 80
+```
+
+**Step 4: Run the command**
+
+```shell
+$ kubectl --context member1 apply -f /tmp/deployments-and-services.yaml
+deployment.apps/nginx-deploy created
+service/nginx-svc created
+deployment.apps/hello-deploy created
+service/hello-svc created
+```
+
+Thus, we can use `member1` as the cluster with existing resources, while `member2` as a bare cluster.
+
+## Tutorials
+
+### Migrate all the resources to Karmada
+
+1）Apply the local file `/tmp/deployments-and-services.yaml` written just now to Karmada control plane too.
+
+**Step 5: Run the command**
+
+```shell
+➜ ✗ kubectl --context karmada-apiserver apply -f /tmp/deployments-and-services.yaml
+deployment.apps/nginx-deploy created
+service/nginx-svc created
+deployment.apps/hello-deploy created
+service/hello-svc created
+```
+
+2）Write a `PropagationPolicy` to the local file `/tmp/pp-for-migrating-deployments-and-services.yaml`.
```

Comment:
This seems to belong to the next step(Step 6). Isn't?

Reply:
Maybe we can remove it. redundant.

Reply:
not unintentional mistake, but the number here is actually confusing, I'll update it